source/icu.net/NativeMethods/NativeMethodsHelper.cs

```diff
@@ -81,7 +82,7 @@ public static IcuVersionInfo GetIcuVersionInfoForNetCoreOrWindows()
 			// If this is false, something went wrong.  These files should have
 			// either been found above or we should have been able to locate the
 			// asset paths (for .NET Core and NuGet v3+ projects).
-			if (!TryGetNativeAssetPaths(context, out var nativeAssetPaths))
+ 			if (!TryGetNativeAssetPaths(context, out var nativeAssetPaths))
```

Comment:
A rouge space snuck in at the start of this line.

Reply:
Fixed